apps/settings/js/panels/hotspot/panel.js

```diff
@@ -23,9 +23,9 @@ define(function(require) {
           hotspotSettingBtn:
             panel.querySelector('#hotspot-settings-section a'),
           hotspotElement:
-            panel.querySelector('input#tethering-wifi-enabled'),
+            panel.querySelector('gaia-switch#tethering-wifi-enabled'),
           usbTetheringElement:
-            panel.querySelector('input#tethering-usb-enabled'),
+            panel.querySelector('gaia-switch#tethering-usb-enabled'),
```

Comment:
I think we can just use "#xxxxx" without gaia-switch in selector because "#id" is specific enough and can make performance better.